core/engine/src/vm/opcode/args.rs

```diff
@@ -0,0 +1,736 @@
+use thin_vec::ThinVec;
```

Comment:
nit (non-blocking): add some unit tests here or an issue to add tests for the unsafe code here.

Still working my reading through this, so I'm familiar with it. :laughing:  It may come up in the future if we have some `unsafe` code here but no unit tests in the module. I wouldn't block merging this on lack of tests, but if none are added, we should at least open up an issue. Plus, adding the unit tests themselves could make a good first issue for someone looking to contribute.

Reply:
Definitely. I will open an issue after the merge.

Reply:
https://github.com/boa-dev/boa/issues/4240